### PR TITLE
Adds github as explicit header menu item. Fixes #873.

### LINKED ISSUE #873: Put 'GitHub' link in menu explicitly
Sorry folks, realized this last night - we don't have 'GitHub' written out in the menu anywhere and that's what folks generally look for to find the repos of projects. I think we may have avoided that before because our situation was ambiguous, but that's not a problem anymore.

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

File: src/components/MainMenu.js [hunk 1]
```diff
@@ -8,16 +8,18 @@ import { Link } from 'react-router-dom';
 import { BetaWarning } from './BetaWarning';
 
 
-const MainMenu = function (props) {
+const MainMenu = function ({ snippets }) {
+  console.log(snippets)
```
Comment (pporfilio) [inline src/components/MainMenu.js]: Did you intend to commit this log statement? If you do want to leave it in, the linter recommends adding a semicolon.

**Review by pporfilio — APPROVED** [review]
Aside from the one line I noted, these changes look stylistically and functionally correct to me. I built locally and tested the links.

File: src/components/MainMenu.js (hunk 1 above)
Comment (knod) [reply src/components/MainMenu.js]: I definitely did not intend to commit it, thank you!

**Comment by knod** [comment]
Re-integrated with dev and github is showing Travis as pending, though Travis says it's done and the tests have passed, so I'm merging this in anyway!

**Comment by knod** [comment]
fyi, that test started failing after I deleted the branch, forgot I deleted it, and re-ran the test. Sorry, folks.

**Pull request merged**